src/lib/helpers.ts

```diff
@@ -73,6 +73,11 @@ export function filterNoMatchReason(
   obj: Partial<KubernetesObject>,
   capabilityNamespaces: string[],
 ): string {
+  // binding deletionTimestamp filter and object deletionTimestamp dont match
+  if (binding.filters && binding.filters.deletionTimestamp && obj.metadata && !obj.metadata.deletionTimestamp) {
```

Comment:
Could this be `binding.filters?.deletionTimestamp`? It'd just be one less conditional (`&&`) to think about while still achieving the same result, I think.

Reply:
Similarly, I think `!obj.metadata?.deletionTimestamp` gets the same result with one less set of `&&`. But I'll defer on if the syntax to express _"Not 'If `metadata` exists then is `deletionTimestamp` truthy?'"_ is easier to read.